src/main/java/virtual_pet/Virtualpet.java

```diff
@@ -0,0 +1,91 @@
+package virtual_pet;
+
+class Virtualpet {
```

Comment:
Be careful of using the wrapper class Integer instead of the primitive data type int.  You should have used just int within this program.  The use of Integer is overkill and would cause performance issues with the simplistic math being done here.